winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,23 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', '$stateParams',
+  function($scope, claimService, $stateParams) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+    $scope.allForms = [
```

Comment:
It should be bundled with the rest of the code when you run `gulp build`, has been working fine for me, is that not true for you?